app/controllers/api/v1/configs_controller.rb

```diff
@@ -0,0 +1,102 @@
+class Api::V1::ConfigsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  before_action :set_config, only: %i[update destroy]
+
+  rescue_from StandardError, with: :handle_standard_error
+  rescue_from ActiveRecord::RecordInvalid, with: :record_invalid
+  rescue_from ActiveRecord::RecordNotUnique, with: :record_not_unique
+  rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
+  rescue_from ActionDispatch::Http::Parameters::ParseError, with: :parse_error
+
+  # GET api/v1/configs/name
+  def show
+    config = Config.friendly.find(params[:id])
+
+    render json: config.body, status: :ok
+  end
+
+  # POST api/v1/configs
+  def create
+    config = current_user.configs.create!(get_config)
+
+    render json: {
+      message: "Config #{config.name} was successfully created.",
+      show_instruction: "If you want to see the saved body, send a GET request to /api/v1/configs/#{config.name}"
+      }, status: :created
```

Comment:
インデントに違和感を感じます。

```suggestion
    render json: {
                 message: "Config #{config.name} was successfully created.",
                 show_instruction: "If you want to see the saved body, send a GET request to /api/v1/configs/#{config.name}"
               }, 
           status: :created
```

こんな感じにすると読みやすくなりそうです。

Reply:
提案いただいた形とは少しだけ違いますが、個人的にしっくりくる形に変えてみました。
https://github.com/pubannotation/textae-configs/pull/13/commits/1ff49b12c37fd0b90e66667e3bdbb70c50c73268

違和感ある場合は再度ご指摘ください :pray: